.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,18 @@
+name: JS Tests
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  js_build_and_test:
+    name: JS Estlin and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
```

Comment:
Shouldn't this match the value in `.nvmrc` - `10.11.0`?

Reply:
`lint-staged requires at least version 10.13.0 of Node, please upgrade` I think we need at least 10.13.0. I am going to bump up our `.nvmrc`

Reply:
Actually, this should probably be done with this issue: https://github.com/Automattic/woocommerce-services/issues/2082

I am going to address it there instead.

Reply:
Oh yeah - always getting that error on the pre-commit hooks :)